deploy/ui/form.json

```diff
@@ -55,6 +55,14 @@
           "jsonPath": "$.spec.useImageTags",
           "originalJsonPath": "$.spec.useImageTags"
         },
+        {
+          "label": "Use Openshift CA",
+          "type": "checkbox",
+          "required": false,
+          "description": "Set true to use the Openshift CA bundle as a truststore. Replaces java's cacert.",
```

Comment:
no, just generates a truststore. we can't create certs against a CA w/o knowing the root CA privateKey. that's still something they'd have to do themselves.

are you sure about the custom truststore not being used instead of cacerts?

Reply:
sry man, I always get confused about the usage of the SSL related properties. 
When the trustStore property is set, the JVM, instead of looking on the default location for JVM's default cacerts will look on the provided location. you're right.

wondering If would make sense to instead set the properties, merge the ocp's on the jvm's one, does it make sense to you? If so, we can add a new task for a future release.

Reply:
this wouldn't work, the operator won't have access to java's cacert as it's bundled within the image. the only way that becomes possible is either with some kind of init container and keytool... or with a buildconfig in openshift. 

i wouldn't recommend either path, and honestly, openshift's ca bundle appears to support more CAs w/ 130 certs... whereas java's `cacert` file only has 93 certs.

Reply:
or we can edit the image itself to search for mounted truststore and append it, maybe. My main concern is when the user needs to provide their own cert but does not have enough permissions to edit the OCP's ca-bundle, but, as discussed below, it can be done later.